src/main/webapp/app/text-assessment/text-assessment.component.html

```diff
@@ -144,10 +145,11 @@ <h4 class="col-12" jhiTranslate="artemisApp.assessment.generalFeedback">General
     </div>
 </ng-container>
 
-<div class="mt-4" *ngIf="hasComplaint && result?.id">
-    <h2 jhiTranslate="modelingAssessmentEditor.reviewComplaint">
-        Review Complaint
-    </h2>
-    <jhi-complaints-for-tutor-form [resultId]="result.id" [isAllowedToRespond]="!isAuthorized" (updateAssessmentAfterComplaint)="onUpdateAssessmentAfterComplaint($event)">
+<div class="mt-4" *ngIf="complaint">
+    <jhi-complaints-for-tutor-form
+        [complaint]="complaint"
+        [isAllowedToRespond]="complaint.complaintType === ComplaintType.COMPLAINT ? !isAuthorized : isAuthorized"
```

Comment:
I think we should rename `isAuthorized` to `isAssessor` here as well, similar to the modeling assessment component.

Reply:
Yes you are right here